README.md

```diff
@@ -62,6 +62,36 @@ This approach should be compatible with all process managers, including
 `strong-pm`. However, we feel that relying on the order of `require` statements
 is error-prone.
 
+### Bind for concurrency
```

Comment:
I think this section should go after `### Use the current context` just before ` ### Use current authenticated user in remote methods`.